scripts/testpop/testpop.go

```diff
@@ -48,6 +48,8 @@ var (
 	faultyPeriod    = flag.Int("faulty-period", 24*60*60*1000, "The lifetime of a faulty member in ms. After that the member becomes a tombstone.")
 	tombstonePeriod = flag.Int("tombstone-period", 5000, "The lifetime of a tombstone member in ms. After that the member is removed from the membership.")
 
+	identity = flag.String("identity", "", "specify an identity")
```

Comment:
I think we should at least say: `specify an identity for this node in the hashring`. Otherwise no point in even having the help text. The fact that it's setting something called identity is already obvious from the name.